boards/launchxl/src/main.rs

```diff
@@ -334,14 +336,19 @@ pub unsafe fn reset_handler() {
     );
     virtual_alarm1.set_client(alarm);
 
+    let etor = static_init!(
+        capsules::rng::Entropy32ToRandom<'static>,
+        capsules::rng::Entropy32ToRandom::new(&cc26x2::trng::TRNG)
+    );
     let rng = static_init!(
-        capsules::rng::SimpleRng<'static, cc26x2::trng::Trng>,
-        capsules::rng::SimpleRng::new(
-            &cc26x2::trng::TRNG,
+        capsules::rng::RngDriver<'static>,
+        capsules::rng::RngDriver::new(
+            etor,
```

Comment:
Does etor stand for entropy_to_random? I think it is better to use the full name here for clarity!